sql/expression/convert.go

```diff
@@ -132,9 +132,13 @@ func (c *Convert) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 		return nil, nil
 	}
 
+	// Should always return nil, and a warning instead
```

Comment:
It sounds like we also need to set the warning? or is that in another part of the code? You can set that warning from the ctx object or the session object from within it if I remember correctly.